### PR TITLE
listview template pull request

### PR DESCRIPTION
**Pull request by attrib75**
## Submission description

Hi, this is my FIRST EVER pull request!  I'm attempting to submit a contribution to Snppts.  I included a listview item template.  Hopefully I'm doing this right.  Let me know if there's anything I can do differently

Screenshot:
screenshot-1573237095029

### CODE REVIEW

**Comment by attrib75** [comment]
I basically don't know what I'm doing lol

File: src/Snppts/Snippets/ListOverviewTemplate.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    public class ListOverviewTemplate : IAmASnippet
+    {
+        public string Slug => "Listview-Item-Template";
+        public string Title => "Listview General Item Template";
+        public string GithubRepoName => "ListViewTemplate";
+        public bool ContainsAndroidSample => true;
+        public bool ContainsiOSSample => true;
+        public bool ContainsUWPSample => false;
+
+        public string Description => "Listview tempalte for displaying item image and details";
```
Comment (felipebaltazar) [inline src/Snppts/Snippets/ListOverviewTemplate.cs]:
```suggestion
        public string Description => "Listview template for displaying item image and details";
```

File: src/Snppts/Snippets/ListOverviewTemplate.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Snppts.Snippets
+{
+    public class ListOverviewTemplate : IAmASnippet
+    {
+        public string Slug => "Listview-Item-Template";
+        public string Title => "Listview General Item Template";
+        public string GithubRepoName => "ListViewTemplate";
```
Comment (felipebaltazar) [inline src/Snppts/Snippets/ListOverviewTemplate.cs]: `GithubRepoName` should contains with your github workspace name, like below:

```suggestion
        public string GithubRepoName => "attrib75/ListViewTemplate";
```
This is used to create a link to your source code, and currently not working:

<p align="center">
	<kbd>
		image
	</kbd>
</p>

Comment (attrib75) [reply src/Snppts/Snippets/ListOverviewTemplate.cs]: Ok, what should I do? change my code and submit another request?

**Comment by attrib75** [comment]
Ok I saw a link to 'commit suggestions' and I clicked that.  I'm assuming it applied the changes to my repo that you suggested, while updating the PR

File: src/Snppts/Snippets/ListOverviewTemplate.cs (hunk 2 above)
Comment (felipebaltazar) [reply src/Snppts/Snippets/ListOverviewTemplate.cs]: Here you can accept my suggestion, but you can make this changes in your PC with your IDE and commit it at your forked repository branch (currently master)..
Are you using command line git? you can use [github desktop](https://desktop.github.com/), it suggest you to send more changes to a specific openned PR

More about contributing to projects:

[Get started](https://help.github.com/en/desktop/contributing-to-projects)
[Committing and reviewing changes](https://help.github.com/en/desktop/contributing-to-projects/committing-and-reviewing-changes-to-your-project)

**Comment by attrib75** [comment]
btw this is also for the community challenge, thanks

**Comment by attrib75** [comment]
If I understand correctly, this has to be reviewed by someone on the snppts team before it would be possible to accept this pull request?

**Review by jfversluis — APPROVED** [review]
That is correct, and I hereby did!

Thanks for your contribution! :)

**Comment by attrib75** [comment]
Wow thank you so much!

**Comment by sthewissen** [comment]
Thanks for contributing to Snppts! Please fill in the following survey to receive your swag :)

https://www.surveymonkey.com/r/88ZQ6G2

**Pull request merged**